continuous_integration/travis/install.sh

```diff
@@ -74,6 +74,8 @@ if [[ $CRICK == true ]]; then
     pip install -q git+https://github.com/jcrist/crick.git
 fi;
 
+conda uninstall --force cryptography
```

Comment:
Why this? Leftover from testing?

Reply:
Yeah, I'll remove that shortly. That's a temporary change to catch any other tests that should have `pytest.importorskip("cryptography")`